apps/mobile-wallet/src/contexts/walletConnect/WalletConnectSessionRequestModal.tsx

```diff
@@ -55,7 +61,8 @@ import { SessionRequestData } from '~/types/walletConnect'
 import { showExceptionToast } from '~/utils/layout'
 import { getTransactionAssetAmounts } from '~/utils/transactions'
 
-interface WalletConnectSessionRequestModalProps<T extends SessionRequestData> extends ModalContentProps {
+interface WalletConnectSessionRequestModalProps<T extends SessionRequestData> {
+  walletConnectClient: SignClient
```

Comment:
Isn't this accessible via `useWalletConnectContext`?

Reply:
Udpated